shared/shared.podspec

```diff
@@ -35,5 +35,5 @@ Pod::Spec.new do |spec|
             SCRIPT
         }
     ]
-
-end
+                
+end
```

Comment:
Please trim the white spaces above and add a blank line at EOF. This file is an auto-generated file this [link](https://stackoverflow.com/questions/24093609/how-can-i-configure-android-studio-to-add-new-line-at-end-of-file) might help

Reply:
This EOF issue has existed since the generated project. 😢  cc @luongvo